test/NuGet.Core.Tests/SynchronizationTestApp/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,23 @@
+﻿//using System.Reflection;
```

Comment:
this looks like a bug in the dnx console app template. I had to comment these out. I'll remove or do another fix as necessary before pushing